qa/tasks/cephfs/filesystem.py

```diff
@@ -778,7 +778,9 @@ def get_daemon_names(self, state=None, status=None):
         """
         mdsmap = self.get_mds_map(status)
         result = []
-        for mds_status in sorted(mdsmap['info'].values(), lambda a, b: cmp(a['rank'], b['rank'])):
+        for mds_status in sorted(
+                mdsmap['info'].values(),
+                lambda a, b: (a['rank'] > b['rank']) - (a['rank'] < b['rank'])):
```

Comment:
Should I ask :-) again to get rid of `cmp` just using _key_ parameter here and later? Like `key=lambda _: _['rank']`.

Reply:
Done :)